src/ServiceControl/SagaAudit/SagaUpdatedHandler.cs

```diff
@@ -43,6 +37,11 @@ public async Task Handle(SagaUpdatedMessage message, IMessageHandlerContext cont
 
         async Task RefreshAuditQueue()
         {
+            if (nextAuditQueueNameRefresh > DateTime.UtcNow)
```

Comment:
What caused this check to be added? It was added as part of the "Cleanup of the SagaUpdatedHandler" commit, but this doesn't seem like cleanup.

Reply:
I did split the commit